articles/libraries/lock-ios/v2/customization.md

```diff
@@ -155,6 +299,16 @@ Color used as the Lock background color.
 }
 ```
 
+### backgroundImage
+
+Image used as the Lock background
+
+```swift
+.withStyle {
+  $0.backgroundColor = LazyImage?
```

Comment:
Oh, and that property name is wrong too :grin:.

I'll look back and see where it came from. Might be from that Styles.swift and I just didn't look closely enough.